plugin-hrm-form/src/permissions/index.ts

```diff
@@ -427,27 +429,39 @@ const setupAllow = <T extends TargetKind>(kind: T, conditionsSets: TKConditionsS
 };
 
 const initializeCanForRules = (rules: RulesFile) => {
+  if (!rules) {
+    throw new Error('Rules not loaded for initializeCanForRules');
```

Comment:
```suggestion
    console.warn('Rules not loaded for initializeCanForRules');
    return () => false;
```

This will allow you to